rcl/test/rcl/test_wait.cpp

```diff
@@ -49,7 +49,8 @@ class CLASSNAME (WaitSetTestFixture, RMW_IMPLEMENTATION) : public ::testing::Tes
     ret = rcl_init_options_init(&init_options, rcl_get_default_allocator());
     ASSERT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
     OSRF_TESTING_TOOLS_CPP_SCOPE_EXIT({
-      EXPECT_EQ(RCL_RET_OK, rcl_init_options_fini(&init_options)) << rcl_get_error_string().str;
+      ret = rcl_init_options_fini(&init_options);
+      EXPECT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
```

Comment:
Is there a functional change to this modification? If not, please keep it as is to minimize the diff.

Same below.